packages/webpack/builder.d.ts

```diff
@@ -3,6 +3,7 @@ declare module '#builder' {
   import type MiniCssExtractPlugin from 'mini-css-extract-plugin'
 
   export const webpack: typeof Webpack
+  export const WebpackBarPlugin: typeof import('webpackbar').default
```

Comment:
can you add an `import type` above like with `MiniCssExtractPlugin`? might that resolve the issue?

Reply:
That was what I did previously, the type was `any`.

Reply:
if so then that is a bug in the types for `webpackbar`, probably